handlers/handler.go

```diff
@@ -77,19 +78,48 @@ func IndexS3Object(s3objectURL string) {
 	log.Printf("Finish to compute hashes for %s", key)
 
 	indexdInfo, _ := getIndexServiceInfo()
-	rev, err := GetIndexdRecordRev(uuid, indexdInfo.URL)
-	if err != nil {
-		log.Println(err)
-		return
+
+	var retries = 0
+	var rev = ""
+
+	for {
+		rev, err = GetIndexdRecordRev(uuid, indexdInfo.URL)
+		if err != nil {
+			retries++
+			time.Sleep(30)
```

Comment:
i'm in favor of putting sleep call into the `if retries == MaxRetries` block so for the last retry we can break from loop immediately but this is minor